### PR TITLE
Seed

### PR DESCRIPTION
**Pull request by Favot**
I finish the seed file and Update the typo on Vinyls view.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,10 +1,310 @@
...
+  status: "declined",
+  total_price: total_price
+)
+
+puts "Finished!"
+
+
+
+
+# total_price: (end_date - start_date) * price_per_day
```
Comment (lisehenry) [inline db/seeds.rb]: missing empty line at the end of the file

**Review by lisehenry — COMMENTED** [review]
ok

File: db/seeds.rb (hunk 1 above)
Comment (Favot) [reply db/seeds.rb]: Noted

**Review by bastarte — APPROVED** [review]
Everything looks perfect

**Pull request merged**